### DISCUSSION TITLE
Troubles with preprocessing of proof

### DISCUSSION THREAD

**Discussion by hextract** [community/community#19475] [GitHub Education] [answered]
I tried many times to send proof of my studing but i always get error while preprocessing (Your image should include the name of your school). But there is a name of my school in doc, but I think trouble causes that it's written on russian. What can I do?

**Comment by melvincwng** [comment, marked as answer]
@h4x4d, instead of using your school ID, try reapplying for the Github Education Student Pack with one or more of the following:

- Class schedule
- Academic transcript
- Letter stating your affiliation with an academic institution or enrollment verification letter

If all fails, contact [Github Education Support](https://support.github.com/contact/education) over here by raising a ticket. They will investigate and get back to you usually within a few working days.

Hope this clarifies.

If this reply is helpful, please 'Mark as answer' so the others are aware that this issue has been addressed.

Thank you, and have a great day ahead 😄

Reference: [Github Docs](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/use-github-for-your-schoolwork/apply-for-a-student-developer-pack)

**Reply by hextract** [reply]
I use my academic transcript now, and I don't how to use schedule.

I writed to github education support with photo of my transcript five days ago and there is no answer now. So I tried to write here if there is people who faced same troubles.

**Reply by melvincwng** [reply]
@h4x4d, I previously helped someone facing the same issue as you. You can check out the discussion thread over [here](https://github.com/github-community/community/discussions/19398).

Other than, my suggestion would be to wait patiently till Github Education Support gets back to you.

Hope this clarifies.

If this reply has helped you, please kindly 'Mark as answer' so the rest are aware this issue is addressed.

Thank you :)

**Reply by nevermind-1** [reply]
I have tried almost all the option but none was successful. what should I do?

**Comment by voidumesha** [comment]
Your image should be one of the following:
Dated school ID 

I am a university student,  How can i fix this

**Reply by voidumesha** [reply]
I have solved that error
